NSSNamespace/Initial_Bangazon_Site
# Es order method try

Details:

- `Products controller` now contains a method, `AddToCart`, that accepts an argument of product id. This method is called on the product detail view.
- First, `AddToCart` queries the database for the active customer’s existing open orders, and if there are none, creates an order and adds a line item with the corresponding product id and order id.
- Otherwise, `AddToCart` adds a line item with that customer's existing open order’s id and product id.
- `Site.js` now contains logic for listener on “Add to Cart” button.
- After a customer adds an item to the cart, she/he is routed back to the homepage (`AllProductsView`)

To Test:

     1. Git fetch

- Checkout branch es-order-method-try
- Select an active customer
- View details for a product
- Click “Add to Cart” button
- Check your database to be sure that if no open order for that customer exists, one was created
- Check your database for a new line item
- Test the method on a customer who has no open order and a customer who does
- Be sure app navigates back to homepage after adding product to cart
